src/svelte/Breadcrumbs.svelte

```diff
@@ -1,18 +1,58 @@
 <script lang="ts">
+  import {onMount} from "svelte";
+
   interface Breadcrumbs {
-    items: Array<{url: string; title: string}>;
-    ariaLabel?: string;
+    items: Array<{url: string; title: string}>
+    ariaLabel?: string
+    showAllAriaLabel?: string
   }
 
   export let breadcrumbs: Breadcrumbs = {items: []};
+  export let loadJs = false
+
   let ariaLabel = breadcrumbs.ariaLabel ?? 'breadcrumbs';
+  let showAllBreadCrumbs = breadcrumbs.showAllAriaLabel ?? 'show all'
+  let spread = true
+  $: breadcrumbData = breadcrumbs.items
```

Comment:
`breadcrumbData` does not need to be responsive
```suggestion
  let breadcrumbData = breadcrumbs.items
```